### PR TITLE
NODE-979: Ask for password if not specified

### PR DESCRIPTION
**Pull request by disfenoid**
[JIRA: https://wavesplatform.atlassian.net/browse/NODE-979](https://wavesplatform.atlassian.net/browse/NODE-979)

### CODE REVIEW

File: src/main/scala/com/wavesplatform/settings/WalletSettings.scala [hunk 1]
```diff
@@ -2,6 +2,59 @@ package com.wavesplatform.settings
...
+    val walletFile =
+      config
+        .as[Option[File]](s"$configPath.file")
+    val seed =
+      config
+        .as[Option[ByteStr]](s"$configPath.seed")
+
+    WalletSettings(walletFile, password, seed)
+  }
+  def readPasswordFromCommandLine(): String = {
```
Comment (potan) [inline src/main/scala/com/wavesplatform/settings/WalletSettings.scala]: Function readPasswordFromCommandLine read a password from console but not from command line. Should we rename it?

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -118,7 +118,7 @@ waves {
     file = ${waves.directory}"/wallet/wallet.dat"
 
     # Password to protect wallet file
-    password = "some string as password"
+//    password = "some string as password"
```
Comment (ismagin) [inline src/main/resources/application.conf]: does this comment work? shouldn't it be # ?

File: src/main/scala/com/wavesplatform/settings/WalletSettings.scala [hunk 3]
```diff
@@ -2,6 +2,59 @@ package com.wavesplatform.settings
...
+        |**********************************
+      """.stripMargin
+
+    try {
+      message.lines
+        .foreach(println)
+
+      System
+        .console()
+        .readPassword("Password > ")
```
Comment (ismagin) [inline src/main/scala/com/wavesplatform/settings/WalletSettings.scala]: is there a way to hide password chars during input?

File: src/test/resources/password.conf [hunk 4]
```diff
@@ -0,0 +1 @@
+some string as password
```
Comment (ismagin) [inline src/test/resources/password.conf]: where is this used?

**Pull request merged**